src/components/GoldenEye/GoldenEye.tsx

```diff
@@ -5,7 +5,10 @@ import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
 import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
-import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import {
+    LoadingOverlayAndCancelButton,
```

Comment:
Check if this is an unused import.

- [ ] Remove unnecessary import.

Thank you!

Reply:
Removed unneccessary imports